symphony-bdk-legacy/symphony-api-client-java/src/test/java/internal/FileHelperTest.java

```diff
@@ -29,4 +30,40 @@ public void should_read_file_from_system() throws IOException {
   public void fail_to_read_file() throws FileNotFoundException {
     FileHelper.readFile(UUID.randomUUID().toString() + ".abc");
   }
-}
\ No newline at end of file
+
+  @Test
+  public void classpath_path_with_slash_on_first() {
+    assertEquals("classpath:/path/to/file.pem", FileHelper.path("classpath:/path/to/", "file.pem"));
+  }
+
+  @Test
+  public void classpath_path_with_slash_on_second() {
+    assertEquals("classpath:/path/to/file.pem", FileHelper.path("classpath:/path/to", "/file.pem"));
+  }
+
+  @Test
+  public void classpath_path_with_two_slashes() {
+    assertEquals("classpath:/path/to/file.pem", FileHelper.path("classpath:/path/to/", "/file.pem"));
+  }
+
+  @Test
+  public void classpath_path_with_no_slashes() {
+    assertEquals("classpath:/path/to/file.pem", FileHelper.path("classpath:/path/to", "file.pem"));
+  }
+
+  @Test
+  public void only_classpath_on_first_slash_on_first() {
+    assertEquals("classpath:/file.pem", FileHelper.path("classpath:/", "file.pem"));
+  }
+
+  @Test
+  public void only_classpath_on_first_slash_on_second() {
+    assertEquals("classpath:/file.pem", FileHelper.path("classpath:", "/file.pem"));
+  }
+
+  @Test
+  public void only_classpath_first_with_no_slash() {
+    assertEquals("classpath:/file.pem", FileHelper.path("classpath:", "file.pem"));
```

Comment:
Please mind that `Paths.get("classpath:", "file.pem")` (=current behavior) will add a path separator in between (`/` on linux/macOS or `\` on Windows)